bentoml/lightgbm.py

```diff
@@ -193,9 +204,10 @@ def num_replica(self) -> int:
 
     # pylint: disable=arguments-differ,attribute-defined-outside-init
     def _setup(self) -> None:
-        self._model = lgb.Booster(
```

Comment:
I think in this case you can just simply use the `load` function to load in the model